### PR TITLE
PRIME-2004 change global renewal constant to 30 days

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

**Review by hogmann — APPROVED** [review]
LGTM

File: prime-dotnet-webapi/Services/Rules/MinorUpdateRules.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ public override Task<bool> ProcessRule(Enrollee enrollee)
             }
 
             TimeSpan diff = enrollee.ExpiryDate.Value - DateTimeOffset.Now;
-            return Task.FromResult(diff > TimeSpan.FromDays(90));
+            return Task.FromResult(diff > TimeSpan.FromDays(30));
```
Comment (mtpultz) [inline prime-dotnet-webapi/Services/Rules/MinorUpdateRules.cs:50]: @james-hollinger request was change renewal periods for enrolments and site registration to 30 from 90 days.  This appears to be the only reference to the 90 renewal period for enrolments on the backend, but wanted to double check.

Comment (james-hollinger) [reply prime-dotnet-webapi/Services/Rules/MinorUpdateRules.cs:50]: Fairly certain this is the only place, though we may need to check emails to see if the 90 day renewal is mentioned anywhere.

**Review by AMiKaH — APPROVED** [review]

**Review by Paahn — APPROVED** [review]
LGTM

File: prime-dotnet-webapi/Services/Rules/MinorUpdateRules.cs (hunk 1 above)
Comment (mtpultz) [reply prime-dotnet-webapi/Services/Rules/MinorUpdateRules.cs:50]: Only other 90 day refs are around Site inactivity so looks like we're good

**Pull request merged**